lib/grpnames.gd

```diff
@@ -73,24 +93,24 @@
 ##  Remark:
 ##   The normal subgroup lattice of G does not contain
 ##   all the information needed for the normal subgroup lattice
-##   of G2 for a decomposition G = G1 x G2. However let 
+##   of G2 for a decomposition G = G1 x G2. However let
 ##   G = G1 x .. x Gr be finest direct product decomposition
 ##   then all Gi are normal subgroups in G. Thus all Gi occur in
-##   the set NormalSubgroups(G) and we may split G recursively 
+##   the set NormalSubgroups(G) and we may split G recursively
 ##   without recomputing normal subgroup lattices for factors.
 ##
 ##  Method to enumerate factorizations given the divisors:
 ##   Consider a strictly increasing chain
 ##     1  <  a1 < a2 < .. < a_n  <  A
-##   of positive divisors of an integer A. 
+##   of positive divisors of an integer A.
 ##   The task is to enumerate all pairs 1 <= i <= j <= n
 ##   such that a_i*a_j = A. This is done by
 ##
-##   i := 1;  
+##   i := 1;
 ##   j := n;
-##   while i <= j do                   
-##     while j > i and a_i*a_j > A do 
-##       j := j-1; 
+##   while i <= j do
+##     while j > i and a_i*a_j > A do
+##       j := j-1;
 ##     end while;
```

Comment:
It's unfortunate that you mixed in removal of whitespace with your other changes, as they obscure the view and make it a tad harder to review your actual changes.

Don't get me wrong, in general I think we should get rid of all trailing spaces (and agree on a uniform way of indenting GAP and C code, and implement that everywhere, etc.), but that should be done explicitly, in dedicated commits.

It's not so hard to resolve this (by either removing these whitespace changes, or moving them into a separate commit) using `git rebase -i`, let me know if you are interested in doing that but need some advice.